src/components/Block/Space.vue

```diff
@@ -37,7 +56,22 @@ const isAdmin = computed(() => {
         <div class="mb-[12px] text-color">
           {{ $tc('members', nbrMembers, { count: _n(nbrMembers) }) }}
         </div>
-        <FollowButton :space="space" />
+        <div class="flex justify-center gap-x-2">
+          <FollowButton :space="space" />
+          <UiSidebarButton
```

Comment:
Maybe we could rename component to `CircleButton` as this is being used in other places too now.